magento/pwa-studio
# [PB-PWA-2469]: Add localization test coverage to cypress

## Description

PR to include automated localization tests in cypress coverage.

Changes:  
 - Adds some missing translations to `fr_FR.json` in `extensions/venia-sample-language-packs`
 - Changed test order to start on Default Store View instead of French since develop is set to  `STORE_VIEW_CODE=default`.

## Related Issue







Closes PWA-2469

## Acceptance





Adds cypress mock data integration test for [PWA-1415](https://jira.corp.magento.com/browse/PWA-1415)



### Verification Stakeholders



### Specification




## Verification Steps





- Check that every test step is covered in cypress automation
- Obs: test steps 12 and 13 cannot be automated due to requiring a second build with different environment variables

#### Test scenario(s) for direct fix/feature




In order
1. Install `@magento/venia-sample-language-packs` by running `yarn venia add -D @magento/venia-sample-language-packs`
2. Set language pack version to `file:../extensions/venia-sample-language-packs`
3. Set env variable `STORE_VIEW_CODE` to `default`
4. Set env variable `USE_STORE_CODE_IN_URL` to `true`
5. Run cypress localization tests against PR environment.
6. Verify that all localization tests pass
7. Verify that all steps mentioned in [PWA-1415](https://jira.corp.magento.com/browse/PWA-1415) are automated in tests (except for steps 12 and 13).
8. Run all cypress tests against PR environment.
9. Verify all cypress tests pass.

#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
